### ISSUE TITLE
Uncaught TypeError: Cannot read property 'join' of null

### ISSUE THREAD

**Issue by zhylninc** [FreeCodeCamp/FreeCodeCamp#6642]
only after landing

//! moment-timezone.js
//! version : 0.5.0
//! author : Tim Wood
//! license : MIT
//! github.com/moment/moment-timezone

vendor-main-00eff8238f.js:formatted:17423Uncaught TypeError: Cannot read property 'join' of null  
  function i(a) {
        var b = a.toTimeString()
          , c = b.match(/\(.+\)/);
        c = c && c[0] ? c[0].match(/[A-Z]/g).join("") : b.match(/[A-Z]{3,5}/g)[0], //error line

**Comment by bugron** [comment]
@zhylninc thanks for posting this issue!
I'm going to close this issue as a duplicate of #6633 but, please, comment there your issue and post a screenshot of your browser's console and Network tab too, thanks.

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***
